src/mgmt/srv_drpc.c

```diff
@@ -50,9 +50,9 @@ pack_daos_response(Mgmt__DaosResp *daos_resp, Drpc__Response *drpc_resp)
 
 	len = mgmt__daos_resp__get_packed_size(daos_resp);
 	D_ALLOC(body, len);
-	if (body == NULL) {
+	if (body == NULL)
 		drpc_resp->status = DRPC__STATUS__FAILED_MARSHAL;
-	} else {
+	else {
```

Comment:
Based on the coding guidelines, I think the original style was correct in this case. The idea being that the usage of curly brackets should match between all the blocks of an if/else statement. So if one of the blocks needs it, all of them do. Not a big deal though.

Reply:
will address in https://github.com/daos-stack/daos/pull/3423 if that's okay

Reply:
https://github.com/daos-stack/daos/pull/3423/commits/74c935f2e90f268555cd656beb4e4593d2d0955b